bitnami/postgresql/templates/_helpers.tpl

```diff
@@ -108,9 +108,9 @@ Get the replication-password key.
         {{- printf "%s" (tpl .Values.global.postgresql.auth.secretKeys.replicationPasswordKey $) -}}
     {{- else if .Values.auth.secretKeys.replicationPasswordKey -}}
         {{- printf "%s" (tpl .Values.auth.secretKeys.replicationPasswordKey $) -}}
-    {{- else -}}
-        {{- "replication-password" -}}
     {{- end -}}
+{{- else -}}
+    {{- "replication-password" -}}
```

Comment:
Shouldn't we also keep the `else` condition that was removed above, in this PR? If we have both, we can basically leave "replication-password" as a fallback value whenever something is not properly configured.

Reply:
Good thinking, I've added the `else` condition again.

Reply:
But then it should be done the same way for `adminPasswordKey` and `userPasswordKey`, right?